cmd/podman/login.go

```diff
@@ -46,7 +46,6 @@ func init() {
 
 	// Podman flags.
 	flags.BoolVarP(&loginOptions.tlsVerify, "tls-verify", "", false, "Require HTTPS and verify certificates when contacting registries")
-	flags.BoolVarP(&loginOptions.GetLoginSet, "get-login", "", false, "Return the current login user for the registry")
```

Comment:
Why are we dropping this? Seems unrelated to the rest of the PR

Reply:
Ditto @mheon's question.  If we are dropping it, we need a man page change.

Reply:
@mtrmac Added this option to github/container/common/pkg/auth, so it is conflicting in podman code and no longer needed.